### ISSUE TITLE
Add a linter

### ISSUE THREAD

**Issue by e-e-e** [RubenVerborgh/SPARQL.js#56]
I just opened this https://github.com/RubenVerborgh/SPARQL.js/pull/55 to remove irregular white spaces, and noticed that this repo currently does not have any linting rules. I might be useful to add to avoid future white space issues, but also help guide contributors with adopting your js style.

**Comment by RubenVerborgh** [comment]
Right, I use linters in all of my other JavaScript libraries indeed. The idea here was that most of the code is automatically generated using Jison in a format I don't control, so that a linter would not really contribute much. In #55, the issue is in `lib/sparql.jison`, which a linter would not catch.

However, I might do some very light linting on the result then, to check for things such as irregular whitespace.

**Comment by e-e-e** [comment]
Cool - thanks for explaining that. I have not looked at jison before, and obviously did not look closely enough. Will need to check it out.
